crates/ruff_linter/src/rules/flake8_bugbear/rules/zip_without_explicit_strict.rs

```diff
@@ -17,7 +17,8 @@ use crate::fix::edits::add_argument;
 /// iterable. This can lead to subtle bugs.
 ///
 /// Use the `strict` parameter to raise a `ValueError` if the iterables are of
-/// non-uniform length.
+/// non-uniform length. If the iterables are intentionally different lengths the
```

Comment:
```suggestion
/// non-uniform length. If the iterables are intentionally different lengths, the
```

Reply:
> If the iterables are intentionally different lengths the parameter should be explicitly set to False.

> If the iterables are intentionally different lengths, the parameter should be explicitly set to False.

I feel like there shouldn't be a comma there, but I'm not 100%.